src/commands/protocols/publish/sender/publish-schedule-messages-command.js

```diff
@@ -6,6 +6,9 @@ class PublishScheduleMessagesCommand extends ProtocolScheduleMessagesCommand {
     constructor(ctx) {
         super(ctx);
         this.operationService = ctx.publishService;
+        this.serviceAgreementService = ctx.serviceAgreementService;
```

Comment:
As far as I see it's not used anywhere

Reply:
It is used in validateBidsForNeighbourhood function (LOC 59), so it was a bug that it was missing.